### PR TITLE
run unittest in python2 and python3

### PR DESCRIPTION
**Pull request by zenghsh3**

### CODE REVIEW

File: .teamcity/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# A dev image based on paddle production image
+
+FROM paddlepaddle/paddle:1.1.0-gpu-cuda9.0-cudnn7
```
Comment (TomorrowIsAnOtherDay) [inline .teamcity/Dockerfile]: Paddle 1.3 has been released, could you update the test environment ?

File: .teamcity/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Run apt-get update
+RUN apt-get install --fix-missing -y cmake
+
+# Prepare packages for Python
+RUN apt-get install --fix-missing -y make build-essential libssl-dev zlib1g-dev libbz2-dev \
+    libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+    xz-utils tk-dev libffi-dev liblzma-dev
+
+# Install python3.6
```
Comment (TomorrowIsAnOtherDay) [inline .teamcity/Dockerfile]: Install python3.6 & pip3.6

Comment (zenghsh3) [reply .teamcity/Dockerfile]: done

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
LGTM ：）

**Pull request merged**